### PR TITLE
enh(swift) add parameter pack keywords

### PR DESCRIPTION
**Pull request by bradleymackey**
Parameter packs are supported in Swift 5.9

Their usage can be seen in [SE-0393](https://github.com/apple/swift-evolution/blob/main/proposals/0393-parameter-packs.md). This essentially just requires a promotion of `each` to a keyword and matching `repeat` and `each` as keywords in generic parameters.

### Changes
- Support `repeat` and `each` parameter pack keywords.

### Checklist
- [x] Added markup tests, or they don't apply here because...
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

File: src/languages/swift.js [hunk 1]
```diff
@@ -302,7 +302,11 @@ export default function(hljs) {
     end: />/,
     contains: [
       ...COMMENTS,
-      TYPE
+      TYPE,
+      {
+        match: /(repeat|each)/,
+        className: "keyword"
```
Comment (joshgoebel) [inline src/languages/swift.js:308]: Lets start using the new `scope` naming... `className` is on the way out.

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
So what is repeat when used outside a generic?

**Comment by joshgoebel** [comment]
Is it not a keyword in those other contexts?

**Comment by bradleymackey** [comment]
It is a keyword in those contexts as well, it's already in the "keywords" list though because it's already used for loops.

```swift
repeat {
    print(number)
    number += 1
} while number <= 20
```

**Comment by joshgoebel** [comment]
Ah that's what I was missing. 😀

File: src/languages/swift.js [hunk 2]
```diff
@@ -302,7 +302,11 @@ export default function(hljs) {
     end: />/,
     contains: [
       ...COMMENTS,
-      TYPE
+      TYPE,
+      {
```
Comment (joshgoebel) [inline src/languages/swift.js:306]: Is there a reason we aren't just using `keywords` here vs a custom match rule?

**Pull request merged**